README.md

```
@@ -1,139 +1,138 @@
-### Usage Monitoring
-- Real-time app usage tracking
-- Background monitoring via WorkManager
-- Usage statistics aggregation
+---
 
-### Intervention System
-- 80% usage threshold notifications
-- Evaluation cycle management
-- Progress tracking across cycles
+## 📁 Project Layout
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)